client/config/crd/groupsnapshot.storage.k8s.io_volumegroupsnapshotclasses.yaml

```diff
@@ -92,3 +92,77 @@ spec:
     served: true
     storage: true
     subresources: {}
+  - additionalPrinterColumns:
+    - jsonPath: .driver
+      name: Driver
+      type: string
+    - description: Determines whether a VolumeGroupSnapshotContent created through
+        the VolumeGroupSnapshotClass should be deleted when its bound VolumeGroupSnapshot
+        is deleted.
+      jsonPath: .deletionPolicy
+      name: DeletionPolicy
+      type: string
+    - jsonPath: .metadata.creationTimestamp
+      name: Age
+      type: date
+    name: v1alpha1
```

Comment:
I don't think we need to keep the v1alpha1 APIs.  We didn't keep the v1alpha1 VolumeSnapshot APIs when we moved it to beta: https://github.com/kubernetes-csi/external-snapshotter/pull/139